src/components/ListItem/ListItem.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'clsx';
+import styles from './ListItem.scss';
+import {Typography} from '~/components';
+
+export const ListItem = ({label, iconStart, iconEnd, isDisabled, variant, className, onClick, onMouseEnter, onMouseLeave, ...props}) => {
+    const cssListItem = classnames(
+        className,
+        styles.listItem,
+        'flexRow',
+        'alignCenter'
+    );
+
+    const handleClick = e => {
+        onClick(e);
+    };
+
+    const handleMouseEnter = e => {
+        onMouseEnter(e);
+    };
+
+    const handleMouseLeave = e => {
```

Comment:
No need to redefine the handlers here, you can directly use onClick() / onMouseEnter() / onMouseLeave()